glass/build.gradle

```diff
@@ -20,10 +20,10 @@ android {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile project(':shared')
-    compile 'com.embarkmobile:zxing-android-minimal:2.0.0@aar'
-    compile 'me.dm7.barcodescanner:zbar:1.5'
-    compile 'com.github.bumptech.glide:glide:4.0.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.0.0'
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation project(':shared')
+    implementation 'com.embarkmobile:zxing-android-minimal:2.0.0@aar'
+    implementation 'me.dm7.barcodescanner:zbar:1.5'
+    implementation 'com.github.bumptech.glide:glide:4.3.1'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.3.1'
```

Comment:
You don't need the compiler if you are not using a generated module, which I don't see, so you can remove this.